YoCode/Program.cs

```diff
@@ -16,9 +16,11 @@ public static class Program
 
         static void Main(string[] args)
         {
+            var outputs = new List<IPrint>();
+            outputs.Add(new WebWriter());
+            outputs.Add(new ConsoleWriter());
```

Comment:
These 3 lines could be 1: `var outputs = new List<IPrint> {new WebWriter(), new ConsoleWriter()};`

Reply:
Could you specify what parameters did you use for this webreport? It should be printing title of the check

Reply:
See my other comment, the problem is not an HTML problem.